pkg/base/linuxutils/lsblk/lsblk.go

```diff
@@ -175,7 +175,7 @@ func (l *LSBLK) SearchDrivePath(drive *api.Drive) (string, error) {
 	for _, l := range lsblkOut {
 		lvid := strings.TrimSpace(l.Vendor)
 		if strings.EqualFold(l.Serial, sn) && strings.EqualFold(lvid, vid) &&
```

Comment:
I think we need to add comments here - why comparison is done this way.

Reply:
Do these comments work?
// Compare drive's sn, vid, pid with lsblk output.
// The hasPrefixIgnoreSpaces function is used to compare pid and lsblk model, accounting for drives that may have
// multiple spaces in their pid, and situations where pid is truncated or limited to 16 digits compared to lsblk model.

or something shorter/longer?

Reply:
Looks good, thank you!